src/components/panes/QueryPane.scss

```diff
@@ -1,5 +1,10 @@
 .PaneBase.QueryPane {
     min-width: 480px;
+
+    @include small-screen {
+        min-width: 100%;
+    }
```

Comment:
Could be replaced by same solution proposed for CallPabe

Reply:
Fixed!